config/vufind/config.ini

```diff
@@ -165,6 +165,17 @@ allowSavedSearches = true
 nonJavascriptSupportEnabled = false
 ; Generator value to display in an HTML header <meta> tag:
 generator = "VuFind 8.0.1"
+; ExpandedHeadTitle style version
+; pre = set the information as prefix before header
+; post = set the information as postfix after header
+; both = set both (prefix and postfix)
+expandedHeadTitle_style = false
```

Comment:
Why camelCase and underscores in the same setting? I'd prefer using one for all settings. Seems to me that there are enough settings that they could all be in their own section such as `[HeadTitle]`, then the actual settings could be simple like `prefix`, `postfix`, `separator`.

Reply:
I think the style setting could be eliminated. Instead the code could check what's defined and use it.